src/corppa/utils/filter.py

```diff
@@ -97,13 +123,33 @@ def main():
     # progress bar is enabled by default; disable if requested
     disable_progress = not args.progress
 
-    output_filename = args.output
+    if not os.path.exists(args.idfile):
+        print(f"Error: idfile {args.idfile} does not exist")
+        exit(-1)
+    elif os.path.getsize(args.idfile) == 0:
+        print(f"Error: idfile {args.idfile} is empty")
+        exit(-1)
+
     # if requested output filename has no extension, add jsonl
+    output_filename = args.output
     if os.path.splitext(output_filename)[1] == "":
```

Comment:
This is assuming an "extension" simply corresponds to the existence of a `.` in the filename. This is probably fine, but will produce `.jsonl` files without the appropriate extension in some cases.

Reply:
I don't think we should worry about that now as it seems unlikely, but if you have specific cases in mind we could document in case we want to revisit.